.goreleaser.yml

```diff
@@ -8,6 +8,8 @@ builds:
       - GO111MODULE=on
     main: ./main.go
     binary: charts-syncer
+    ldflags:
+      - -X github.com/bitnami-labs/charts-syncer/cmd.Version={{.Version}}
```

Comment:
goreleaser will take the version from the git tag

Reply:
nice!